samples/web/content/apprtc/constants.py

```diff
@@ -22,3 +22,30 @@
 RESPONSE_UNKNOWN_CLIENT = 'UNKNOWN_CLIENT'
 RESPONSE_DUPLICATE_CLIENT = 'DUPLICATE_CLIENT'
 RESPONSE_SUCCESS = 'SUCCESS'
+
+BIGQUERY_URL='https://www.googleapis.com/auth/bigquery'
```

Comment:
@jiayliu will have to comment but I tend to agree, although i'm torn about the BIGQUERY_* ones.

Regardless, if we want to move this I will file a separate PR